package.json

```diff
@@ -5,28 +5,32 @@
   "main": "index.js",
   "scripts": {
     "postinstall": "typings install --overwrite --save",
-    "test": "karma start"
+    "test": "karma start karma.unit.js",
+    "coverage": "karma start karma.coverage.js"
   },
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "awesome-typescript-loader": "^3.1.2",
```

Comment:
Should we fix the awesome-typescript-loader? Some weeks they release a minor update that broke, not sure about the rest of packages.